torch/distributions/__init__.py

```diff
@@ -58,7 +58,7 @@
 sample therefore becomes differentiable. The code for implementing the pathwise
 derivative would be as follows::
 
-    params = policy_network(state)
+    arg_constraints = policy_network(state)
     m = Normal(*params)
```

Comment:
nit: should be `arg_constraints` I think.

Reply:
gosh thanks. Fixed.